File: app/src/main/java/com/example/kirill/retrofittry/Interfaces/GetCourses.java
package com.example.kirill.retrofittry.Interfaces;

import com.example.kirill.retrofittry.Parsers.Course;
import com.example.kirill.retrofittry.Parsers.Courses;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface GetCourses {
    @GET("courses")
    Call<Courses> all();

    @GET("courses/{id}")
    Call<Course> select(@Path("id") int id);
}

File: app/src/main/java/com/example/kirill/retrofittry/interfaces/GetCourses.kt
package com.example.kirill.retrofittry.interfaces

import com.example.kirill.retrofittry.Parsers.Course
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface GetCourses {
    @GET("courses")
    fun all(): Call<List<Course>>

    @GET("courses/{id}")
    fun select(@Path("id") id: Int): Call<Course>
}